<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Trilho1">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>40</y>
      <width>140</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border: 1px solid #000;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="TremLaranja">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>30</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: orange</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="TremVermelho">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: red</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Trilho2">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>139</y>
      <width>140</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border: 1px solid #000;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="TremAzul">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: blue</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Trilho3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>139</y>
      <width>140</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border: 1px solid #000;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="TremPreto">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>326</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Trilho4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>238</y>
      <width>140</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border: 1px solid #000;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>Trilho1</zorder>
   <zorder>Trilho2</zorder>
   <zorder>Trilho3</zorder>
   <zorder>Trilho4</zorder>
   <zorder>TremAzul</zorder>
   <zorder>TremPreto</zorder>
   <zorder>TremVermelho</zorder>
   <zorder>TremLaranja</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
